xml/System.Reflection.Metadata/MemberReference.xml

```diff
@@ -257,8 +258,8 @@
         <ReturnType>System.Reflection.Metadata.EntityHandle</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>MethodDef, ModuleRef,TypeDef, TypeRef, or TypeSpec handle.</summary>
```

Comment:
Can we improve this? It should with Gets the...

Reply:
```suggestion
        <summary>Gets an entity handle instance. If the <see cref="P:System.Reflection.Metadata.EntityHandle.IsNil" /> property is <see langword="true" />, the returned handle will have default values.</summary>
```

Reply:
Updated. @mairaw does this sound better?

Reply:
Actually, look at the value. I restored this summary to "To be added" as you suggested in the other comment.